daemons/upload-checksum-daemon/requirements.txt

```diff
@@ -6,3 +6,4 @@ dcplib
 sqlalchemy
 alembic
 psycopg2-binary
+psycopg2
```

Comment:
why do we need psycopg2 and psycopg2-binary in requirements.txt?

Reply:
removed